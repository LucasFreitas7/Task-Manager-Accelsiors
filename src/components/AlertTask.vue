<template>
<div>
    <v-alert
      v-if="success"
      dense
      text
      type="success"
    >
      {{ message }}
    </v-alert>
    <v-alert
    v-else
    dense
    outlined
    type="error"
  >
    {{ message }}
  </v-alert>
</div>
</template>
  
  <script>
    export default {
      name: 'AlertTask',
      props:{
        message:{
            type: String,
            default: ''
        },
        success:{
            type: Boolean,
            default: false
        }
      }
}
  </script>
