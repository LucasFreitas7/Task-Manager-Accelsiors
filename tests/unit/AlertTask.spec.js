import { shallowMount } from '@vue/test-utils'
import AlertTask from '@/components/AlertTask.vue'
describe('AlertTask.vue', () => {

  it('Snapshot test', () => {
    const wrapper = shallowMount(AlertTask)
    expect(wrapper.html()).toMatchSnapshot()
  })
  it('when pass props message, she must be renderized', () => {
    const wrapper = shallowMount(AlertTask, {
      propsData: { message: 'test' }
    })
    expect(wrapper.find('v-alert').text()).toBe('test')
  })
})
