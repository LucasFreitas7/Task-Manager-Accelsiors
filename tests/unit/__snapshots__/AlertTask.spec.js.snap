// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertTask.vue Snapshot test 1`] = `
<div>
  <v-alert dense="" outlined="" type="error">

  </v-alert>
</div>
`;
